backend/src/kitconcept/intranet/behaviors/configure.zcml

```diff
@@ -20,10 +20,24 @@
       />
 
   <plone:behavior
-      name="kitconcept.sitecustomization"
-      title="Customizations for sites/subsites"
+      name="kitconcept.sitecustomization.header"
+      title="Header customizations for sites/subsites"
       description="Set of fields to customize sites and subsites"
-      provides=".customizations.ISiteCustomizationSettings"
+      provides=".customizations.ISiteHeaderCustomizationSettings"
+      />
+
+  <plone:behavior
+      name="kitconcept.sitecustomization.theme"
+      title="Theme colors customizations for sites/subsites"
+      description="Set of fields to customize sites and subsites"
+      provides=".customizations.ISiteThemeCustomizationSettings"
+      />
+
+  <plone:behavior
+      name="kitconcept.sitecustomization.footer"
+      title="Footer customizations for sites/subsites"
+      description="Set of fields to customize sites and subsites"
+      provides=".customizations.ISiteFooterCustomizationSettings"
       />
```

Comment:
I was planning to name the behaviors voltolighttheme.header, voltolightheme.colors, voltolighttheme.footer

Reply:
I don't feel strongly about the names! But to avoid confusion they should probably have a prefix which matches the name of the package where they are defined, in the end.

Reply:
Yes, good point. I will amend them.